stingray/tests/test_crossspectrum.py

```diff
@@ -685,36 +682,6 @@ def test_make_crossspectrum_diff_lc_counts_shape(self):
                 AssertionError, match="No GTIs are equal to or longer than segment_size"
             ):
                 Crossspectrum(self.lc1, lc_)
-            with pytest.raises(
-                StingrayError, match="Light curves do not have same number of time bins"
-            ):
-                Crossspectrum(self.lc1, lc_, legacy=True)
-
-    def test_make_crossspectrum_diff_lc_stat(self):
-        lc_ = copy.deepcopy(self.lc1)
-        lc_.err_dist = "gauss"
-
-        with pytest.warns(UserWarning) as record:
-            cs = Crossspectrum(self.lc1, lc_, legacy=True)
-        assert np.any(["different statistics" in r.message.args[0] for r in record])
```

Comment:
These tests that are removed here seem to test what happens if you have a different error distribution. Is that no longer allowed in the new version (since it needs the legacy interface)? Or is it still allowed, in which case maybe we need different tests here?

Reply:
Oh, good catch!

Reply:
Done